client/admin-pointers.js

```diff
@@ -0,0 +1,26 @@
+/*global wcSevicesAdminPointers, ajaxurl */
+import jQuery from 'jquery';
```

Comment:
jQuery code built by Webpack. Now that's something I didn't expect to see today. I like it 😄